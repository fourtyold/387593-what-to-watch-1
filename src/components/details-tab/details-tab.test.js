import React from "react";
import renderer from "react-test-renderer";

import DetailsTab from "./details-tab.jsx";

const options = {
  film: {
    director: `director`,
    starring: [`actor1`, `actor2`, `actor3`],
    runTime: 100,
    genre: `genre`,
    released: 2000
  }
};

it(`Details tab correctly renders`, () => {
  const tree = renderer.create(
      <DetailsTab
        film={options.film}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
