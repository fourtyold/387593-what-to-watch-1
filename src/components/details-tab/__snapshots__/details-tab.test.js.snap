// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details tab correctly renders 1`] = `
Array [
  <div
    className="movie-card__text movie-card__row"
  >
    <div
      className="movie-card__text-col"
    >
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Director
        </strong>
        <span
          className="movie-card__details-value"
        >
          director
        </span>
      </p>
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Starring
        </strong>
        <span
          className="movie-card__details-value"
        >
          actor1

          actor2

          actor3

        </span>
      </p>
    </div>
    <div
      className="movie-card__text-col"
    >
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Run Time
        </strong>
        <span
          className="movie-card__details-value"
        >
          1h 40m
        </span>
      </p>
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Genre
        </strong>
        <span
          className="movie-card__details-value"
        >
          genre
        </span>
      </p>
      <p
        className="movie-card__details-item"
      >
        <strong
          className="movie-card__details-name"
        >
          Released
        </strong>
        <span
          className="movie-card__details-value"
        >
          2000
        </span>
      </p>
    </div>
  </div>,
  ";",
]
`;
